test/CMakeLists.txt

```diff
@@ -1456,3 +1447,17 @@ add_custom_test(test_regression_opencl_float_mi100 GFX908_ENABLED VEGA_DISABLED
     COMMAND	${ENVS_REGRESSION_ISSUE_1012} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --cmode conv --pmode default --group-count 1 --input 64,  512, 28, 28 --weights 128, 512, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 ${ARGS_REGRESSION_ISSUE_1012}
     COMMAND	${ENVS_REGRESSION_ISSUE_1012} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --cmode conv --pmode default --group-count 1 --input 64,  64,  56, 56 --weights 256, 64,  1, 1 --pads_strides_dilations 0 0 1 1 1 1 ${ARGS_REGRESSION_ISSUE_1012}
 )
+
+if(MIOPEN_TEST_GFX1030)
+    if(WORKAROUND_ISSUE_1053 AND MIOPEN_TEST_ALL)
+        list(APPEND SKIP_TESTS test_lrn_test)
+    endif()
```

Comment:
It's too late to add `test_lrn_test` to `SKIP_TESTS` here. Please move this thiss back (near line 190).

Reply:
Yes, this is a terrible mistake which I essentially wasted one peaceful day (I usually do these tests on weekend). Thanks!

Reply:
cmake syntax is a nightmare ;)